packages/typescript/test/member-expression.test.tsx

```diff
@@ -143,6 +153,164 @@ describe("with refkeys", () => {
     `);
   });
 
+  it("handles nullish symbols correctly", () => {
```

Comment:
Might be good to note that this test is deliberately producing invalid references (it's just easiest to test with VarDeclaration).